### PR TITLE
Flux 2.0 GA Announcement

### PR DESCRIPTION
**Pull request by somtochiama**
Preview: https://deploy-preview-1570--fluxcd.netlify.app/blog/2023/07/flux-ga/

### CODE REVIEW

File: content/en/blog/2023-07-17-june-update/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+## Flux 2.0 and General Availability!
+
+![FluxCD community stats](infographic-featured.png)
+
+> Best Buy: Tahir Raza, Staff Engineer - Cloud & Platform Engineering: 
+> “GitOps practices and Flux has elevated our engineering: code infra
+> as software, eliminate human intervention, accelerate lead time for changes -
+> without compromising security requirements.”
```
Comment (stefanprodan) [inline content/en/blog/2023-07-17-june-update/index.md:20]: Maybe move this below, as the first phrase is used for social media embeds.

File: content/en/blog/2023-07-17-june-update/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+Enterprises such as SAP, RingCentral, Best Buy, Volvo and many others 
+also trust Flux to be an essential part of their Kubernetes journey.
+
+> Safaricom PLC: Winnie Gakuru, DevSecOps Engineer II: "Safaricom PLC 
+> rovides mobile telephony, mobile money transfer, consumer electronics, e-commerce,
+> cloud computing, data, music streaming, and fiber optic services to the Kenyan
+> Market predominantly and to the wider East Africa. So, Flux has been an essential
+> part of critical areas such as deployment frequency, standardization, and security,
+> among other GitOps capabilities that help us to be competitive. We are excited
+> about Flux 2.0 and the project's continued maturity.
```
Comment (stefanprodan) [inline content/en/blog/2023-07-17-june-update/index.md:46]:
```suggestion
> Safaricom PLC provides mobile telephony, mobile money transfer, consumer electronics,
> e-commerce, cloud computing, data, music streaming, and fiber optic services to the Kenyan
> Market predominantly and to the wider East Africa. So, Flux has been an essential
> part of critical areas such as deployment frequency, standardization, and security,
> among other GitOps capabilities that help us to be competitive. We are excited
> about Flux 2.0 and the project's continued maturity.
>
> Winnie Gakuru, DevSecOps Engineer II at Safaricom PLC
```

I propose this format for all quotes.

File: content/en/blog/2023-07-17-june-update/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+---
+author: somtochiama
+date: 2023-07-18 20:30:00+00:00
+title: June 2023 Update
```
Comment (stefanprodan) [inline content/en/blog/2023-07-17-june-update/index.md:4]:
```suggestion
title: Announcing Flux 2.0 GA
```

File: content/en/blog/2023-07-17-june-update/index.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+description: "Flux 2.0 and General Availability!"
+url: /blog/2023/07/june-2023-update/
```
Comment (stefanprodan) [inline content/en/blog/2023-07-17-june-update/index.md:6]:
```suggestion
url: /blog/2023/07/flux-ga/
```

File: content/en/blog/2023-07-17-june-update/index.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+tags: [monthly-update]
```
Comment (stefanprodan) [inline content/en/blog/2023-07-17-june-update/index.md:7]:
```suggestion
tags: [announcement]
```

**Review by stefanprodan — COMMENTED** [review]
The quotes placement feels wrong to me, how about creating a section for those and writing a little intro for it?

File: content/en/blog/2023-07-17-june-update/index.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+> without compromising security requirements.”
+>
+> Tahir Raza, Staff Engineer - Cloud & Platform Engineering at Best Buy
+
+It’s, therefore, no surprise that the open-source Flux project has reached many
+great achievements: continued developmental improvements and thought leadership
+such as microservices architecture, security-first design, multi-tenancy, OCI and
+Cosign support, and more. as part of this journey, Flux is a graduated project in the Cloud
+Native Computing Foundation and is the GitOps tool that companies such as Microsoft,
+AWS, GitLab, D2iQ, and more trust to deliver GitOps to their customers.
```
Comment (aryan9600) [inline content/en/blog/2023-07-17-june-update/index.md:37]:
```suggestion
Intending to be the best GitOps tool available, the Flux project has evolved into a mature and trustworthy software. During its evolution, Flux has accomplished several goals such as low resource consumption by adopting a microservices architecture, safe multi-tenancy through its security-first design and support for bleeding edge innovation by having first-class support for technologies like OCI and Cosign.
We are proud to see that Flux is one of the few CNCF graduated projects and the GitOps tool that companies such as Microsoft, AWS, GitLab, D2iQ, and more trust to deliver GitOps to their customers. 
```

File: content/en/blog/2023-07-17-june-update/index.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+> or mature their Kubernetes journey. Keeping great company with users (the likes
+> of Amazon AWS, D2iQ, Microsoft Azure, VMware, Weaveworks, GitLab, Volvo, SAP,
+> Xenit and many more) keeps me motivated for the future innovations and growth for Flux.”
+>
+> Stefan Prodan, Principal Developer Experience Engineer at Weaveworks, Flux maintainer and
+> Flagger creator
+
+Release Cadence: Flux will have at least three minor releases in a year
+following the Kubernetes release cadence. The release will happen roughly
+two weeks after the Kubernetes release. Two weeks timeline can be adjusted if more
```
Comment (aryan9600) [inline content/en/blog/2023-07-17-june-update/index.md:108]:
```suggestion
two weeks after the Kubernetes release. The two weeks timeline can be adjusted if more
```

Comment (aryan9600) [inline content/en/blog/2023-07-17-june-update/index.md:108]:
```suggestion
two weeks after a new Kubernetes release. Two weeks timeline can be adjusted if more
```

File: content/en/blog/2023-07-17-june-update/index.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+time is needed for testing the Kubernetes version.
```
Comment (aryan9600) [inline content/en/blog/2023-07-17-june-update/index.md:109]:
```suggestion
time is needed for testing compatibility with the new Kubernetes version.
```

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**